src/sec_related_rates.mbx

```diff
@@ -527,35 +598,44 @@
             <p>At what rate is the top of the ladder sliding down the side of the house when the base is:</p>
 
             <ol>
-              <li><p>1 foot from the house?</p></li>
-
-              <li><p>10 feet from the house?</p></li>
-
-              <li><p>23 feet from the house?</p></li>
-
-              <li><p>24 feet from the house?</p></li>
+              <li><p>1 foot from the house?</p>
+                <p><var name="$a" width="10"/></p>
+              </li>
+              <li><p>10 feet from the house?</p>
+                <p><var name="$b" width="10"/></p>
+              </li>
+              <li><p>23 feet from the house?</p>
+                <p><var name="$c" width="10"/></p>
+              </li>
+              <li><p>24 feet from the house?</p>
+                <p><var name="$d" width="10"/></p>
+              </li>
             </ol>
           </statement>
-          <solution>
-            <ol>
-              <li><p><m>0.04</m> ft/s</p></li>
-
-              <li><p><m>0.458</m> ft/s</p></li>
-
-              <li><p><m>3.35</m> ft/s</p></li>
-
-              <li><p>Not defined; as the distance approaches 24, the rates approaches <m>\infty</m>.</p></li>
-            </ol>
-          </solution>
         </webwork>
       </exercise>
+<!--Exercise 10: Winch that boat in-->
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
+            <var name="$a">
+              <static>30.59</static>
+            </var>
+            <var name="$b">
+              <static>36.1</static>
+            </var>
+            <var name="$c">
+              <static>301</static>
+            </var>
+            <var name="$d">
+              <!--FUNGUS: Not sure what to put here. Text answer.-->
```

Comment:
Hmm. Maybe just comment out everything having to do with this part of the question for now.